beacon-chain/core/validators/validator.go

```diff
@@ -166,10 +166,10 @@ func ActivateValidator(state *pb.BeaconState, idx uint64, genesis bool) (*pb.Bea
 
 	state.ValidatorRegistry[idx] = validator
 
-	log.WithFields(logrus.Fields{
-		"index":           idx,
-		"activationEpoch": validator.ActivationEpoch - params.BeaconConfig().GenesisEpoch,
-	}).Info("Validator activated")
+	// log.WithFields(logrus.Fields{
+	// 	"index":           idx,
+	// 	"activationEpoch": validator.ActivationEpoch - params.BeaconConfig().GenesisEpoch,
+	// }).Info("Validator activated")
```

Comment:
Commented these out because they would horribly spam during the benchmarks. Can we find another way to display this info that's less intrusive for internal uses?